cmd/registry/cmd/config/config_test.go

```diff
@@ -100,108 +100,116 @@ func TestConfig(t *testing.T) {
 
 	cmd := setCommand()
 	cmd.SetArgs([]string{"test", "test"})
-	_, err = capture(cmd, "")
-	if err.Error() != `Config has no property "test".` {
+	want := `Config has no property "test".`
+	if err := cmd.Execute(); err.Error() != want {
```

Comment:
Can we compare error values instead of hardcoding a string? In other words, can we make sure this test will pass even if we change the error message? Maybe that means defining `type MissingPropertyError struct{field string}` and checking if `err != MissingPropertyError{"test"}`.

Reply:
sure

Reply:
sure